static/js/main.js

```diff
@@ -30,6 +30,7 @@ function showEmptyForm(event) {
     // Remove id & class, copied from emptyForm
     newForm.removeAttribute("id");
     newForm.classList.remove("empty_form");
+    newForm.classList.add(metadataType);
```

Comment:
I'm not seeing where this added class is being used, when I inspect or view source it doesn't appear, except for the explicitly added `class = "description">` from line 197 in `edit_item.html`.  Can you please clarify how this is used?  It also conflicts with the comment & lines of code above it, which remove things, not add them.

Not necessarily wrong, but needs clarification if nothing else.

Reply:
`class = "description"` matters due to the first new CSS rule, which hides all `<tr>`s of optional metadata on file-level items unless they have that class. I've added a comment addressing this and separating it from the removals above.